schedule/yast/sle/flows/default_sle15sp4_x86.yaml

```diff
@@ -1,41 +1,52 @@
 ---
-name: ncurses_interactive_installation
-description: >
-  Interactive installation with ncurses (textmode).
-vars:
-  DUD_ADDONS: sdk
-  YUI_REST_API: 1
-schedule:
+# Default ordered sequence of steps for sle15sp4 with maintenance updates
+bootloader:
   - installation/bootloader_start
+setup_libyui:
   - installation/setup_libyui
+product_selection:
   - installation/product_selection/install_SLES
+self_update: []
+license_agreement:
   - installation/licensing/accept_license
+registration:
   - installation/registration/register_via_scc
+extension_module_selection:
   - installation/module_registration/register_extensions_and_modules
   - installation/module_registration/add_additional_regcodes
   - installation/module_registration/import_untrusted_gnpupg_key
+additional_products:
   - installation/add_on_product_installation/add_additional_products
+add_on_product:
   - installation/add_on_product/skip_install_addons
-  - installation/system_role/select_role_text_mode
+system_role:
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+suggested_partitioning:
   - installation/partitioning/accept_proposed_layout
+clock_and_timezone:
   - installation/clock_and_timezone/accept_timezone_configuration
+local_user:
   - installation/authentication/use_same_password_for_root
   - installation/authentication/default_user_simple_pwd
+software:
```

Comment:
this should not be a default, default is to have all patterns, we will use this test module where makes sense to override.